ovos_plugin_manager/utils/__init__.py

```diff
@@ -82,14 +82,25 @@ def find_plugins(plug_type=None):
     else:
         plugs = plug_type
     for plug in plugs:
-        for entry_point in pkg_resources.iter_entry_points(plug):
+        for entry_point in _iter_entrypoints(plug):
             try:
                 entrypoints[entry_point.name] = entry_point.load()
+                LOG.debug(f"Loaded plugin entry point {entry_point.name}")
             except Exception as e:
                 LOG.exception(f"Failed to load plugin entry point {entry_point}")
     return entrypoints
 
 
+def _iter_entrypoints(plug_type):
```

Comment:
Could we add some test case for this? Maybe a test skill branch we install that specifies an exact OPM version older than this one?

Reply:
lets do this in another PR before the next stable release, i want to get this fix out asap as this makes it really painful to use some plugins